tests/playwright/sanity/modules/nested-tabs/helper.ts

```diff
@@ -151,3 +151,77 @@ export async function isTabTitleVisible( context: Page | Frame, positionIndex: n
 
 	return true;
 }
+
+export async function deleteItemFromRepeater( editor: EditorPage, widgetID: string ) {
+	// Arrange
+	const deleteItemButton = editor.page.locator( '.elementor-repeater-row-tool.elementor-repeater-tool-remove .eicon-close' ),
+		nestedItemTitle = editor.getPreviewFrame().locator( `.elementor-element-${ widgetID } .e-n-tab-title` ),
+		nestedItemContent = editor.getPreviewFrame().locator( `.elementor-element-${ widgetID } .elementor-widget-container > .e-n-tabs > .e-n-tabs-content > .e-con` ),
+		numberOfTitles = await nestedItemTitle.count(),
+		numberOfContents = await nestedItemContent.count();
+
+	// Act
+	await deleteItemButton.last().click();
+
+	await editor.getPreviewFrame().waitForSelector( `.elementor-element-${ widgetID }` );
+
+	// Assert
+	await expect.soft( nestedItemTitle ).toHaveCount( numberOfTitles - 1 );
+	await expect.soft( nestedItemContent ).toHaveCount( numberOfContents - 1 );
+}
+
+export async function addItemFromRepeater( editor: EditorPage, widgetID: string ) {
```

Comment:
deleteItemFromRepeater and addItemFromRepeater uses the same logic. If you put 
export async function addItemFromRepeater( widgetID: string, button: Locator, title: locator, content: locator, count: number} ) it can be converted to one function